pyproject.toml

```diff
@@ -26,7 +26,9 @@ classifiers = [
 dependencies = [
     "nomad-lab>=1.3.3.dev86",
     "xmltodict==0.13.0",
-    "fairmat-readers-xrd~=0.0.3"
+    "fairmat-readers-xrd~=0.0.3",
+    "pynxtools>=0.6.0",
+    "pynxtools-xrd>=0.0.2"
```

Comment:
I see, but we can still add the same dependency in the `pynxtools` repo. Since nomad-measurements does not directly use pynxtools-xrd, I don't see a reason to make it a dep here.

Reply:
We have it there, but still developer (at least you guys) needs to know which version is needed while debugging. End of the day you will get `pynxtools-xrd` via `pynxtools` it is just the wording of the package nothing else. However, I will fix it.